spec/fixtures/configuration_examples.rb

```diff
@@ -15,36 +15,20 @@
   include_context 'base configuration'
 
   let(:test_target) { :test_target }
+  let(:env_file)    { '.env.rspec' }
   let(:test_target_config) do
     {
       scale: 1,
       command: ['sleep', 'infinity'],
-      env_files: File.join(FIXTURES_PATH, '.env.test')
+      env_files: File.join(FIXTURES_PATH, env_file)
     }
   end
 
   before(:each) do
     Broadside.configure do |c|
+      c.ecs.cluster = 'cluster'
```

Comment:
i got rid of the `shared_context 'ecs configuration'` because it was useless; no one ever used anything except the `full configuration` context in specs, which is now this context (called `deploy configuration`)